nginx/README.md

```diff
@@ -66,3 +67,9 @@ docker run -d --name "my-awesome-nginx" \
 And then access the PWA with `http://b2b.<your-fully-qualified-machine-name>:4199`
 
 If your DNS is not set up correctly, you have to use something like _dnsmasq_ (Linux) or _Acrylic DNS Proxy_ (Windows), or just ask your local network administrator.
+
+## Deactivating Features
+
+It is possible to deactivate certain features of the nginx by setting environment variables. Deactivating the cache, can be done by setting the environment `DISABLE_CACHE` to _any_ value. If the cache should also be disabled for static resources, the pagespeed feature also has to be switched off as it caches optimized images individually.
+
+Features can be supplied in the folder `nginx/features`. A file named `<feature>.conf` in included if the environment variable `DISABLE_<feature>` (checked case in-sensitive) is not found. If the feature is disabled, an optional file `<feature>-off.conf` is included in the configuration. The feature name must match be all word-characters (letters, numbers and underscore).
```

Comment:
This paragraph was written a bit too fast. ;)
"A file named `<feature>.conf` **in** included if the", "The feature name must **match be all** word-characters".